docs/query-language.md

```diff
@@ -645,3 +647,41 @@ Sum(field="diskusage")
 ```
 
 * Result is the sum of all values (total size of all repositories in kilobytes, here), plus the count of columns.
+
+### Other Operations
+
+#### Options
+
+**Spec:**
+
+```
+Options(<CALL>, columnAttrs=<BOOL>, excludeColumns=<BOOL>, excludeRowAttrs=<BOOL>, shards=[UINT ...])
+```
+
+**Description:**
+
+Modifies the given query as follows:
```

Comment:
I think we should change these to be something like:
```
argument: action the argument affects
```
So, for example:
* `columnAttrs`: Include column attributes in the result (Default: `false`).
* `excludeColumns`: Include column IDs in the result (Default: `false`).
* `excludeRowAttrs`: Include row attributes in the result (Default: `false`).
* `shards`: Run the query on only the given shards. By default, the query is executed on all shards.

Reply:
Re: `shards`. Wouldn't it be more correct to say *for all shards* or maybe *against all shards*?

Reply:
My order of preference for those words would be:
```
1) on
2) against
3) for
```
I guess you could get really specific and say something like "Run the query using only the data from the given shards. By default, the entire data set (i.e. data from all shards) is used.

Reply:
ok, updated